libs9s/s9srpcreply.cpp

```diff
@@ -347,6 +347,46 @@ S9sRpcReply::users()
     return theList;
 }
 
+S9sVariantList
+S9sRpcReply::dbVersions()
+{
+    S9sVariantList  theList;
+    
+    if (contains("db_versions"))
+        theList = operator[]("db_versions").toVariantList();
+    else if (contains("db_versions"))
+        theList << operator[]("db_versions");
+
+    return theList;
+}
+
+S9sVariantList
+S9sRpcReply::clusterTypes()
+{
+    S9sVariantList  theList;
+    
+    if (contains("cluster_types"))
+        theList = operator[]("cluster_types").toVariantList();
+    else if (contains("cluster_types"))
+        theList << operator[]("cluster_types");
+
+    return theList;
+}
+
+S9sVariantList
+S9sRpcReply::vendors()
+{
+    S9sVariantList  theList;
+    
+    if (contains("vendors"))
+        theList = operator[]("vendors").toVariantList();
+    else if (contains("vendors"))
```

Comment:
same contains("vendors") condition in if and its esle if, why it is so?

Reply:
you are actually right, I copied it from S9sRpcReply::clusters() I will change that